src/index.ts

```diff
@@ -137,17 +137,18 @@ Total Paths:** \t${paths.length}\n`;
 	while (true) {
 		await loop.step();
 		await loop.reset();
-
-		if (startupTime - Date.now() + botConfig.signOfLife * 60000 <= 0) {
+		const now = Date.now();
+		if (startupTime - now + botConfig.signOfLife * 60000 <= 0) {
 			timeIt++;
 			const mins = (botConfig.signOfLife * timeIt) % 60;
 			const hours = ~~((botConfig.signOfLife * timeIt) / 60);
-			startupTime = Date.now();
+			startupTime = now;
 			const message = `**chain:** ${chainOperator.client.chainId} **wallet:**  **status:** running for ${
 				loop.iterations
 			} blocks or ${hours === 0 ? "" : hours + " Hour(s) and "}${mins} Minutes`;
+			loop.clearIgnoreAddresses();
```

Comment:
can't we put this in a different place?

Reply:
I searched for a place where it gets called on a regular but long timeframe. This function should just prevent a buildup of addresses that send one or more txs that fail and is abondened after that